README

```diff
@@ -30,14 +30,18 @@ LICENSE:
         LGPL-2.1-or-later for all code, exceptions noted in LICENSES/README.md
 
 REQUIREMENTS:
-        Linux kernel >= 3.15
-        Linux kernel >= 4.2 for unified cgroup hierarchy support
-        Linux kernel >= 4.10 for cgroup-bpf egress and ingress hooks
-        Linux kernel >= 4.15 for cgroup-bpf device hook
-        Linux kernel >= 4.17 for cgroup-bpf socket address hooks
-        Linux kernel >= 5.3 for bounded-loops in BPF program
-        Linux kernel >= 5.4 for signed Verity images support
-        Linux kernel >= 5.7 for BPF links and the BPF LSM hook
+        Linux kernel ≥ 4.4
+                     ≥ 4.5 for pids controller in cgroup v2
+                     ≥ 4.6 for cgroup namespaces
+                     ≥ 4.10 for cgroup-bpf egress and ingress hooks
+                     ≥ 4.15 for cgroup-bpf device hook and cpu controller in cgroup v2
+                     ≥ 4.17 for cgroup-bpf socket address hooks
+                     ≥ 5.3 for bounded-loops in BPF program
+                     ≥ 5.4 for signed Verity images support
+                     ≥ 5.7 for BPF links and the BPF LSM hook
+
+        Versions below 4.15 have significant gaps in functionality and are not
+        recommended for use with this version of systemd.
```

Comment:
if this is changed, also needs updating of `KERNEL_BASELINE_VERSION` in src/basic/def.h